scanner/src/main/kotlin/provenance/NestedProvenanceStorage.kt

```diff
@@ -58,10 +58,10 @@ interface NestedProvenanceStorage {
     fun readNestedProvenance(root: RepositoryProvenance): NestedProvenanceResolutionResult?
 
     /**
-     * Put the resolution [result] for the [root] provenance into the storage. If the storage already contains an entry
-     * for [root] it is overwritten.
+     * Write the resolution [result] for the [root] provenance into the storage. If the storage already contains an
+     * entry for [root] it is overwritten.
```

Comment:
out of scope: I've recently mentioned (slack) the constraint violations in the logs which occur when the entry already exists. Without looking into it, I guess that it's actually not overwritten but ignored in the Postgres based implementation.

Reply:
@fviernau I'm only aware of possible constraint violations related to storing scan results, but not for provenance resolution results.